modules/deployments-rolling-strategy.adoc

```diff
@@ -60,3 +60,9 @@ These parameters allow the deployment to be tuned for availability and speed. Fo
 
 Generally, if you want fast rollouts, use `maxSurge`. If you have to take into account resource quota and can accept partial unavailability, use
 `maxUnavailable`.
+
+[WARNING]
+====
+The default setting for `maxUnavailable` is `1` for all the machine config pools in {product-title}. Do not change this value to `3` for the control plane pool.
```

Comment:
> Do not change this value to `3` for the control plane pool.


In the control plane pool the value should not be changed at all. If we change it to 2 we will have problems too if we apply any config.

I would say this instead:

> Do not change this value for the control plane pool.

Reply:
@sinnykumari @yuqi-zhang  Please, could you confirm this?

At least I have never been able to successfully configure maxUnavailable=2 in a master pool.

Reply:
Yes, I think so, it's best not to parallel any control plane pool updates.

I think in theory if you had > 3 control plane nodes, there might be enough for quorum that you are allowed to set maxUnavailable > 1, but I don't know if that's allowed in practice, so for safety, I think always update 1 master at a time

Reply:
The request emphasized not setting this value to three specifically. it's not recommended to change it and set it to `2`, but it sounds like setting it to `3` has a much worse consequence for the customer, based on this [comment](https://issues.redhat.com/browse/OCPBUGS-24188?focusedId=23756655&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23756655)

Reply:
I don't know off the top of my head if there's any difference between `2` and `3` in this case. I'm fine with the phrasing either way, but I think from the MCO perspective it's safer to not leave any vagueness here and recommend against changing the value entirely.

Happy to follow up as well in case there are other locations, doesn't have to be part of this PR

Reply:
Agree, it's good to explicitly just say something like changing maxUnavailable value of master pool is not recommended.

Reply:
I've updated the wording to try and provide the best of both worlds 🙂